com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/MacOSBundler.java

```diff
@@ -181,6 +184,18 @@ public void bundleApplication(Project project, Platform platform, File bundleDir
 
         BundleHelper.throwIfCanceled(canceled);
 
+        if (variant.equals(Bob.VARIANT_DEBUG))
+        {
+            logger.info("Adding debug entitlements");
+            File entitlementsFile = BundleHelper.copyResourceToTempFile("resources/macos/entitlements-debug.plist");
+            Result r = Exec.execResult("codesign", "-f", "-s", "-", "--entitlements", entitlementsFile.getAbsolutePath(), appDir.getAbsolutePath());
+            if (r.ret != 0) {
+                throw new IOException(new String(r.stdOutErr));
+            }
+        }
+
+        BundleHelper.throwIfCanceled(canceled);
+
```

Comment:
What if user has its own `entitlements` specified in `game.project`?

Reply:
But that's only for iOS? We don't have that field on macOS.

Reply:
ah, true. my bad